apps/ejabberd/src/mod_offline.erl

```diff
@@ -276,7 +272,6 @@ determine_amp_strategy(Strategy, _, _, _, _) ->
 %% Description: Initiates the server
 %%--------------------------------------------------------------------
 init([Host, AccessMaxOfflineMsgs]) ->
-    random:seed(os:timestamp()),
```

Comment:
`random` is deprecated now, so I've replaced `random:uniform` with `rand:uniform` in Riak backend. This new module inits proper seed on first `uniform` call (as opposed to hardcoded default seed in `random`).